### PR TITLE
Added pretty print functionality

### PR DESCRIPTION
**Pull request by pc-alves**
fixes issue #97

### CODE REVIEW

File: cmd/eskip/doc.go [hunk 1]
```diff
@@ -77,10 +77,11 @@ const (
 	prependFileUsage    = "prepend filters from a file to each patched route"
 	appendFiltersUsage  = "append filters to each patched route"
 	appendFileUsage     = "append filters from a file to each patched route"
+	prettyUsage         = "prints routes in a more readable format"
 
 	// command line help (1):
 	help1 = `Usage: eskip <command> [media flags] [--] [file]
-Commands: check|print|upsert|reset|delete
+Commands: check|print|upsert|reset|delete|patch
```
Comment (szuecs) [inline cmd/eskip/doc.go]: This seems not to be relevant for this PR, but maybe correct to fix it.

File: eskip/string.go [hunk 2]
```diff
@@ -84,12 +84,14 @@ func (r *Route) predicateString() string {
...
 }
 
-func (r *Route) filterString() string {
+func (r *Route) filterString(pretty bool) string {
 	var sfilters []string
 	for _, f := range r.Filters {
 		sfilters = appendFmt(sfilters, "%s(%s)", f.Name, argsString(f.Args))
 	}
-
+  if pretty {
```
Comment (szuecs) [inline eskip/string.go]: Please use gofmt to format your code.

File: cmd/eskip/doc.go (hunk 1 above)
Comment (aryszka) [inline cmd/eskip/doc.go]: yes, thanks for the fix

File: eskip/string.go [hunk 3]
```diff
@@ -103,15 +105,23 @@ func (r *Route) backendString() string {
 
 // Serializes a route expression. Omits the route id if any.
 func (r *Route) String() string {
+	return r.Print(false)
+}
+
+func (r *Route) Print(pretty bool) string {
```
Comment (aryszka) [inline eskip/string.go]: i like it. Can we please do the same for `func String(routes ...*Route) string` to stay a bit consistent?

File: eskip/string_test.go [hunk 4]
```diff
@@ -94,6 +94,56 @@ func TestDocString(t *testing.T) {
 		`route2: Path("/some/path") -> "https://www.example.org"`)
 }
 
+func TestPrintNonPretty(t *testing.T) {
+	for i, item := range []struct {
+		route    string
+		expected string
+	} {
+		{
+			"route1: Method(\"GET\") -> filter(\"expression\") -> <shunt>",
```
Comment (aryszka) [inline eskip/string_test.go]: you can also use backtick strings for this, so you don't need to escape doubleqoutes

File: eskip/string.go (hunk 3 above)
Comment (pc-alves) [inline eskip/string.go]: Not sure what you mean.
a) Having a `Print` function for multiple routes, with a pretty option?
b) Using the `Print` function in `func String(routes ...*Route) string`?

Comment (aryszka) [inline eskip/string.go]: sorry for not being completely clear. I meant a). See `func (r *Route) String()` string and `func String(routes ...*Route) string`. There could be a `func Print(routes ...*Route) string`, too.

Comment (pc-alves) [inline eskip/string.go]: Check the latest commit (7aa655c). I think I got it right.

**Comment by aryszka** [comment]
:+1:

**Comment by aryszka** [comment]
thanks, @pc-alves !

**Pull request merged**